<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userPageDialog</class>
 <widget class="QDialog" name="userPageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Main Page</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-797</y>
        <width>1236</width>
        <height>1593</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Welcome</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="userName">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>23</pointsize>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="text">
            <string>'s covid record</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="genelateButton">
           <property name="text">
            <string>Genelate QR cord</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="currentDateUserPage">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Your details</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0" rowspan="2">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Gender</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="genderLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="2" rowspan="2">
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Note</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="noteLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Mobile number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mobileLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>User name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1" rowspan="2">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Date of birth</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="birthdayLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>User ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="userIdLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Vaccine status</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="vaccineLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Email address</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="emailLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Address</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="addressLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Covid-19 test  result</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QTableView" name="testTableView">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>120</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="tabletTracking">
               <bool>true</bool>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Vaccine record</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QTableView" name="vaccineTableView">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>120</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QCommandLinkButton" name="covidCommandLinkButton">
                <property name="font">
                 <font>
                  <family>Tahoma</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Covid Vaccines</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit_5">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="plainText">
               <string>Everyone in New Zealand aged 5 or over can get a free COVID-19 vaccination. It does not matter what your visa or citizenship status is.

Children aged 6 months to 4 years can get the paediatric Pfizer vaccine if they are at higher risk of severe illness from COVID-19.

Pfizer is the preferred COVID-19 vaccine for use in New Zealand.

If you are 18 years of age or older and either cannot get the Pfizer vaccine or want a different option, you can get the Novavax vaccine.

You will be given the option to choose your vaccine type when you book. The Novavax vaccine is not available at all vaccination sites.

Whichever vaccine you get, you will need 2 doses. COVID-19 vaccines are free.

COVID-19 vaccines used in New Zealand.
For more information please click on above link</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCommandLinkButton" name="BoosteCommandLinkButton">
                <property name="font">
                 <font>
                  <family>Tahoma</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Booster</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit_3">
              <property name="plainText">
               <string>An additional Pfizer BA.4/5 COVID-19 bivalent booster is now available for people who have completed a primary vaccination course (for most people this is 2 doses).

You can get this booster, regardless of how many boosters you have had, if you are:

aged 30 and over or
at higher risk of severe illness from COVID-19.
It will need to be at least 6 months since your last COVID-19 vaccine or a positive COVID-19 test.

Having COVID-19 does not provide the same level of immunity as getting vaccinated. We also know that your protection from the primary course (this is usually 2 doses) of the vaccine decreases over time.

To keep your immunity levels high, stay up to date with your vaccinations — including boosters. This will lower your chances of getting very sick from COVID-19 and ending up in hospital.

For boosters, we use the Pfizer BA.4/5 bivalent vaccine. This is an updated vaccine targeting Omicron. COVID-19 vaccines are free.

The bivalent vaccine produces an immune response against the original virus and Omicron variants. Specifically the BA.4 and BA.5 subvariants.

Current evidence indicates the bivalent vaccine is more effective against Omicron subvariants than earlier vaccines.

If you do not want the bivalent vaccine, you can ask for the original Pfizer vaccine when you arrive for your vaccination.
For more information about Boosters, please click above link.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCommandLinkButton" name="updateCommandLinkButton">
                <property name="font">
                 <font>
                  <family>Tahoma</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Updates</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit_2">
              <property name="plainText">
               <string>3 Feb 2023
From 1 March the Pfizer bivalent vaccine will replace the existing booster. From 1 April, an additional booster will be available for those eligible.
All New Zealanders aged 30 and over will be able to access the new COVID-19 bivalent booster from April as part of the Government’s plan to keep Kiwis safe and take pressure off our health system.

The new bivalent vaccine will replace the existing Pfizer booster and is considered likely to be more effective against Omicron subvariants. Bivalent vaccines work by combining 2 strains of a virus, which prompts the body to create antibodies against both strains, providing a greater level of protection.

From 1 April, anyone 30 years or over will be eligible to receive the bivalent vaccine, as long as it has been at least 6 months since their last COVID-19 booster or positive COVID-19 test.

People at higher risk of severe illness from COVID-19 will also be able to receive an additional booster, regardless of how many doses they have previously had.

As well as broadening the criteria from April, from 1 March anyone currently eligible for a COVID-19 booster will be able to get the new bivalent vaccine.

We know vaccination is the best protection against the virus, and these actions will provide added protection to a larger number of adult New Zealanders. The latest report from the Ministry of Health indicated 8,220 cases over the previous week so the virus is definitely still circulating out there.

We are also planning for older New Zealanders and those most at risk of getting sick to get a flu jab at the same time as their COVID-19 booster. This will be part of a ‘one and done’ approach to help people stay well this winter.

Because it may be a while since your previous boosters, now is a good time to check your records on My Covid Record.

Log in to My Covid Record(external link)

Vaccination against COVID-19 remains free.  

Getting boosted is important for your health and keeping you out of hospital this winter.
For more updates, please click above link.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QCommandLinkButton" name="symptomCommandLinkButton">
                <property name="font">
                 <font>
                  <family>Tahoma</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Symptoms</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit_6">
              <property name="plainText">
               <string>OVID-19 symptoms
Symptoms can include one or more of the following:

a new or worsening cough
sneezing and runny nose
a fever
temporary loss of smell or altered sense of taste
sore throat
shortness of breath
fatigue/feeling of tiredness.
Less common symptoms may include:

diarrhoea
headache
muscle pain or body aches
nausea
vomiting
malaise — a general feeling of discomfort, illness or unease
chest pain
abdominal pain
joint pain
confusion or irritability.
These less common symptoms almost always occur with one or more of the common symptoms.

These symptoms do not necessarily mean you have COVID-19. The symptoms are like other illnesses that are much more common, such as colds and flu.

Shortness of breath is a sign of possible pneumonia and requires immediate medical attention.

If you have any of these symptoms, contact your doctor, Healthline on 0800 358 5453 or your iwi health provider.

Time for symptoms to appear
If you get symptoms with Omicron, you are likely to get them within 3 days of infection. But it could take up to 8 days after infection.

Sometimes people may have COVID-19, but not have any symptoms.

Some people with COVID-19 can pass it on to others 1 or 2 days before showing symptoms.
For more information about Symptoms, please click above link.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
